src/libmongoc/tests/test-mongoc-background-monitoring.c

```diff
@@ -166,6 +168,26 @@ _server_changed (const mongoc_apm_server_changed_t *event)
    bson_mutex_unlock (&tf->mutex);
 }
 
+#define TV \
+   "{ 'processId': { '$oid': 'AABBAABBAABBAABBAABBAABB' }, 'counter': 1 }"
+
+bool
+auto_respond_polling_ismaster (request_t *request, void *ctx)
```

Comment:
This auto responder is also used in responding to RTT ismasters for the new streaming tests.